### PR TITLE
Add initial doc to dao branch

### PR DESCRIPTION
**Pull request by PraveshMoelchand**
This PR contains the initial doc files for our functionality in the [common](https://github.com/emieldesmidt/trustchain-superapp/tree/doc/common) folder and [voting](https://github.com/emieldesmidt/trustchain-superapp/tree/doc/voting) submodule. Click on the links to see the rendered READMEs.

Note that the docs will be updated as the code and functionality will be updated.

### CODE REVIEW

File: voting/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# TrustChain Voter
+This submodule of the TrustChain Super App enables a front-end for using the functionality that is provided in the `common/util/Votinghelper` API. For documentation on the API itself, please see 
+
+This module currently contains the following functionality:
+- Retrieve and show list of current proposals including tally
+- Cast a vote on a proposal
+- Create a new proposal based in Yes / No voting mode or threshold voting mode
+
+## Context
+*TODO: In what setting this module is used*
```
Comment (emieldesmidt) [inline voting/README.md:10]: Should't this be a comment?

File: voting/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<img src="https://raw.githubusercontent.com/emieldesmidt/trustchain-superapp/doc/doc/voting/all-proposals.png" width="180"> <img src="https://raw.githubusercontent.com/emieldesmidt/trustchain-superapp/doc/doc/voting/new-proposals.png" width="180">
+
+## Creating a new proposal
+A user can create a new proposal by clicking on the `create proposal` button in the bottom right corner. A dialog will pop up in which the user can enter a name for the proposal and switch between Yes/No voting (left) and threshold voting (right).
+
+<!-- TODO: change file path when merging to Tribler repo -->
+<img src="https://raw.githubusercontent.com/emieldesmidt/trustchain-superapp/doc/doc/voting/create-yes-no-proposal.png" width="180"> <img src="https://raw.githubusercontent.com/emieldesmidt/trustchain-superapp/doc/doc/voting/create-threshold-proposal.png" width="180">
+
+### Yes/No voting mode
+In the Yes/No voting mode the result of the proposal is determined by counting the number of Yes and No votes.
```
Comment (emieldesmidt) [inline voting/README.md:25]: maybe mention that there are drawbacks to using this, without explicitly explaining them

File: voting/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# TrustChain Voter
+This submodule of the TrustChain Super App enables a front-end for using the functionality that is provided in the `common/util/Votinghelper` API. For documentation on the API itself, please see
```
Comment (emieldesmidt) [inline voting/README.md:2]: Perhaps good to explain that this is for demo purposes as well

File: common/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# TrustChain common module
+This module contains functionality and resources that is used by multiple other submodules of the TrustChain Super App.
```
Comment (emieldesmidt) [inline common/README.md:2]: Refer to the voting module for a demo

Comment (PraveshMoelchand) [reply common/README.md:2]: I did that in the VotingHelper section of this README

File: voting/README.md (hunk 1 above)
Comment (PraveshMoelchand) [reply voting/README.md:10]: You’re right!

File: voting/README.md (hunk 2 above)
Comment (PraveshMoelchand) [reply voting/README.md:25]: Good suggestion! Which drawbacks do you think we should list?

File: common/README.md (hunk 4 above)
Comment (emieldesmidt) [reply common/README.md:2]: Fair enough!

File: voting/README.md (hunk 3 above)
Comment (PraveshMoelchand) [reply voting/README.md:2]: Will do!

**Comment by emieldesmidt** [comment]
Thanks for the work @PraveshMoelchand

**Pull request merged**